ReactiveCocoa/Swift/Property.swift

```diff
@@ -138,46 +136,22 @@ public func <~ <T>(property: MutableProperty<T>, producer: SignalProducer<T, NoE
 	let propertyDisposable = property.producer.start(completed: {
```

Comment:
I think you could move this into the `startWithSignal` closure below, then simply dispose of `signalDisposable` upon completion. That would avoid the need to allocate an additional `CompositeDisposable`.